src/main/java/core/basesyntax/FileDataReader.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class FileDataReader {
+    private static String SPLIT_BY_SPACES = " ";
```

Comment:
I would suggest rename this constant to WHITESPACE_SEPARATOR or just WHITESPACE to describe the value of this variable